src/main_menu.cpp

```diff
@@ -702,6 +702,13 @@ bool main_menu::new_character_tab()
         vSubItems.push_back( pgettext( "Main Menu|New Game", "Play Now!  (<F|f>ixed Scenario)" ) );
         vSubItems.push_back( pgettext( "Main Menu|New Game", "Play <N|n>ow!" ) );
     }
+    std::vector<std::string> hints;
+    hints.push_back( "Allows you to fully customize points pool, scenario, and character's profession, stats, traits, skills and other parameters." );
```

Comment:
No translation?

Reply:
Translation happens below, on line 722.

Reply:
These strings are not marked for translation. And `_()` will do nothing.